examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -542,8 +558,10 @@ bool Commands::DecodeArgumentsFromBase64EncodedJson(const char * json, std::vect
     VerifyOrReturnValue(parsedArguments, false, ChipLogError(chipTool, "Error while parsing json."));
     VerifyOrReturnValue(jsonArguments.isObject(), false, ChipLogError(chipTool, "Unexpected json type, expects and object."));
 
-    auto mandatoryArguments = GetArgumentsFromJson(command, jsonArguments, false /* addOptional */);
-    auto optionalArguments  = GetArgumentsFromJson(command, jsonArguments, true /* addOptional */);
+    std::vector<std::string> mandatoryArguments;
+    std::vector<std::string> optionalArguments;
+    VerifyOrReturnValue(GetArgumentsFromJson(command, jsonArguments, false /* addOptional */, mandatoryArguments), false);
```

Comment:
Not for this PR, but eventually we may want to consider that boolean is often not ideal in mixed arguments ... since you have to specify `addOptional` anyway for readability, we could enum this to `OptionalArguments::kInclude / OptionalArguments::kSkip` or similar.